src/picture.rs

```diff
@@ -1,13 +1,12 @@
 use ::libc;
 use crate::src::cdf::CdfContext;
 use crate::src::msac::MsacContext;
+use crate::{stderr,errno_location};
```

Comment:
In some of the `use`s, there are spaces missing after the commas.  It would be fixed once we do a `cargo fmt` later on once we're reading to do that, though.